app/src/main/java/org/p2p/wallet/send/ui/main/SendPresenter.kt

```diff
@@ -705,6 +713,14 @@ class SendPresenter(
         return fees.feeInSol to fees.feeInPayingToken
     }
 
+    private fun showError() {
```

Comment:
can't agree with this naming
presenter can't show error
it can handle it, or request view to show error

Reply:
ok